transcom/mymove
# B-19191: Email Template not locating all duty location names

## [Agility ticket B-19191](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19191)
## Steps to test
1. Fire up the server with the following command.
```
EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run
```
2. Fire up the client as well.
Note: make sure you set the Customer's personal email to an address you can check
3. Create a Move/Shipment with Chelmsford, MA 01824 (or some other location that doesn't automatically register a duty location other than the City, State, Zip) for the origin location.
4. You should receive the email which is triggered when move details are submitted. The email should contain the Origin address information (as seen below).
![Screenshot 2024-08-27 at 4 28 11 PM](https://github.com/user-attachments/assets/6e608a3a-4938-47ed-ab33-d4bdbb3bdf12)
